pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+requires = [
+    "setuptools>=60",
+    "setuptools-scm>=8.0",
+]
+
+[tools.setuptools.dynamic]
+version = {attr = "ome_zarr.__version__"}
+
+[project]
+name = "ome-zarr"
+readme = "README.rst"
+dynamic = ["version"]
+description="Implementation of images in Zarr files."
+
+requires-python = ">3.8"
+
+dependencies = [
+    "tifffile",
```

Comment:
```suggestion
```
This doesn't seem to be an actual dependency?

Reply:
:+1: This is a remainder from when napari was a dependency in this repo (see https://github.com/ome/ome-zarr-py/pull/54)